readme.txt

```diff
@@ -248,6 +248,7 @@ The plugin is produced by [Modern Tribe Inc](http://m.tri.be/18uc).
 * Fix - Add warning if a ticket has stock management turned off in the related WooCommerce product, but has capacity enabled for the ticket(thanks Isaiah Baker and others for highlighting this) [91471]
 * Fix - Make sure the correct menu parent is expanded on the admin when visiting the list of attendees [93057]
 * Fix - Fixes the missing notification on the email and removes the notification when there's none [99979]
+* Fix - Prevent to overwrite the start date of a ticket if was already set [99601]
```

Comment:
grammar change — change to `Prevent overwriting the start date of a ticket if it was already set`